cmd/node/config/systemSmartContractsConfig.toml

```diff
@@ -4,9 +4,9 @@
     MinUnstakeTokensValue = "10000000000000000000" #10eGLD
     UnBondPeriod = 250
     MinStepValue = "100000000000000000000"
-    StakeEnableEpoch = 2
-    StakingV2Epoch = 2
-    DoubleKeyProtectionEnableEpoch = 2
+    StakeEnableEpoch = 1
+    StakingV2Epoch = 1
```

Comment:
I would revert (value 2) the following flags: `StakingV2Epoch`, `DoubleKeyProtectionEnableEpoch`, `DelegationManagerSystemSCConfig.EnabledEpoch`,  `DelegationSystemSCConfig.EnabledEpoch`

Reply:
done